packages/core/src/model/nodes/ChatNode.ts

```diff
@@ -1047,6 +1047,18 @@ export class ChatNodeImpl extends NodeImpl<ChatNode> {
               }
             }
 
+            if (err.status === 408) {
+              if (retriesLeft) {
+                context.onPartialOutputs?.({
+                  ['response' as PortId]: {
+                    type: 'string',
+                    value: 'OpenAI API timed out, retrying...',
+                  },
+                });
+                return;
+              }
+            }
+
```

Comment:
Considered doing something like

```
  if (err.status === 429 || err.status === 408) {
    if (retriesLeft) {
      const errorMessage = {
        429: 'OpenAI API rate limit exceeded, retrying...',
        408: 'OpenAI API timed out, retrying...',
      }[err.status];

      context.onPartialOutputs?.({
        ['response' as PortId]: {
          type: 'string',
          value: errorMessage,
        },
      });
      return;
    }
  }
  ```

but while there's just the two of them, just duplicating the code seemed cleaner to me